src/lxc/tools/lxc_autostart.c

```diff
@@ -504,5 +508,10 @@ int main(int argc, char *argv[])
 	toss_list(cmd_groups_list);
 	free(containers);
 
+	if (failed == count)
+		return -1;	/* Total failure */
```

Comment:
Hm, what I gather from #2523 this should be:
```
if (failed == count)
        exit(1);
else if (failed > 0)
        exit(2);
```

Reply:
Cool, thanks for the review.  Will fix and re-push.